udata/commands/db.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+from os.path import join
+from pkg_resources import resource_isdir, resource_listdir, resource_string
+
+from flask import current_app
+
+from mongoengine.connection import get_db, DEFAULT_CONNECTION_NAME
+
+from udata.commands import submanager, green, yellow, cyan, purple
+
+log = logging.getLogger(__name__)
+
+
+m = submanager(
+    'db',
+    help='Databse related operations',
+    description='Handle all databse related operations and maintenance'
+)
+
+# A migration script wrapper recording the stdout lines
+SCRIPT_WRAPPER = '''
+function(plugin, filename, script) {{
+    var stdout = [];
+    function print() {{
+        var args = Array.prototype.slice.call(arguments);
+        stdout.push(args.join(' '));
+    }}
+
+    {0}
+
+    db.migrations.insert({{
+        plugin: plugin,
+        filename: filename,
+        date: ISODate(),
+        script: script,
+        output: stdout.join('\n')
+    }});
+
+    return stdout;
+}}
+'''
+
+# Only record a migration script
+RECORD_WRAPPER = '''
+function(plugin, filename, script) {{
+    db.migrations.insert({{
+        plugin: plugin,
+        filename: filename,
+        date: ISODate(),
+        script: script,
+        output: 'Recorded only'
+    }});
+}}
+'''
+
+# Date format used to for display
+DATE_FORMAT = '%Y-%m-%d %H:%M'
```

Comment:
Why not [isoformat](https://docs.python.org/2/library/datetime.html#datetime.datetime.isoformat)?